### DISCUSSION TITLE
山东大学

### DISCUSSION THREAD

**Discussion by T3041** [CollegesChat/university-information#171] [Q&A]
想问一下山东大学的两个校区的宿舍情况：是不是上床下桌，有没有独立卫浴

**Comment by yyzsx** [comment]
坐标山东大学兴隆山校区，学校现在是网络免费网速还行有些时候很卡有些时候很块，洗澡要钱2元一次左右用的是软件，在寝室楼下没有寝室没有厕所和洗澡的地方，上厕所每层楼都有，食堂小贵平均一中午12到15元，选修课麻烦要换校区有免费校车但是要刷卡。寝室上床下桌四个人，听说有些中心的是六个人。老师基本上很负责，上课都是很努力想让你明白。环境很好，学校绿化是做了研究的，但是兴隆山和软件园很偏，中心，趵突泉和洪家楼在市区。

**Comment by yyzsx** [comment]
学校18周结课19周考试，所有校区大一大二要跑三千（青岛校区干的），大一要晚自习，一般晚上不查寝，但是老师有可能要查旷课，有充电桩在楼下，济南就兴隆山有共享电车，其他不知道

**Comment by bigdogdan** [comment]
想问一下山东大学机械博士宿舍环境怎么样
